corpora/csip/metadata/amdsec/CSIP31/test-case.xml

```diff
@@ -1,24 +1,85 @@
 <!-- Root element for an individual test case, allows these to be wrapped into
 XML lists -->
 <testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd"
-  testable="UNKNOWN">
+  testable="TRUE">
   <!-- Unique ID for the test case -->
   <id specification="E-ARK CSIP" version="2.0-DRAFT" requirementId="CSIP31"/>
   <!-- URL references to requirements for convenient lookup -->
   <references>
     <reference requirementId="CSIP31" URL="http://earkcsip.dilcis.eu/#CSIP31"/>
   </references>
   <!-- The full text of the requirement. -->
-  <requirementText>If administrative / preservation metadata is available, it must be described using the administrative metadata section (amdSec) element.
-It is possible to transfer metadata in a package using just the descriptive metadata sectiond/or adminstrative metadata section.
-</requirementText>
+  <requirementText><description>If administrative / preservation metadata is available, it must be described using the administrative metadata section (amdSec) element.
```

Comment:
There is a wrong description-element

Reply:
It is actually valid, looking at the new schema, I think we are supposed to use it like this. @koit